generators/app/templates/static/scss/utilities/_utilities.content-editor.scss

```diff
@@ -0,0 +1,366 @@
+.u-content-editor,
+.rte {
+    word-break: break-word;
+
+    > h2,
+    > h3,
+    > h4,
+    > h5,
+    > h6 {
+        margin-bottom: var(--s-30);
+
+        &:last-child {
+            margin-bottom: 0;
+        }
+
+        + h2,
+        + h3,
+        + h4,
+        + h5,
+        + h6 {
+            margin-top: var(--s-60);
+        }
+
+        > u {
+            text-decoration: underline;
+        }
+
+        > a {
+            text-decoration: underline;
+            transition: getTransition(default, ease-in-quad);
+
+            @include hover {
+            }
+        }
+    }
+
+    > p,
+    > ul,
+    > ol {
+        @include b0;
+
+        margin-bottom: var(--s-15);
+
+        &:last-child {
+            margin-bottom: 0;
```

Comment:
maknuo bi to

Reply:
ajmo ovaj cijeli file komentirati na nekom sastanku pa da imamo usporedno tvoj i moj fajl, kak smo se dogovorili na meetu ako ti je to ok?

Reply:
moze